### DISCUSSION TITLE
Automate saving by URL instead of tab

### DISCUSSION THREAD

**Discussion by kwhitefoot** [gildas-lormeau/SingleFile#719] [Ideas]
Is it possible to activate saving by URL so that it would not be necessary to enable it each time I visit a page?

Or is there a way of using a User Script to achieve the same end?

**Comment by gildas-lormeau** [comment]
You can do this by following these steps:
 - open the "Auto-settings rules" panel in the settings of SingleFile
 - check the option "display 'Auto-save profile' column" below the empty table
 - add a row in the table with `*` as "URL" and select `Disabled` in the "Auto-save profile" column
 - add a row in the table with `example.com` (e.g. the site you want to auto-save) as "URL" and let the other fields unchanged
 - Select in the context-menu of SingleFile "Auto-save > auto-save all tabs"

**Reply by gildas-lormeau** [reply]
This can also be done with a separate profile to make things more convenient when adding a new site to autosave, see https://github.com/gildas-lormeau/SingleFile/issues/233#issuecomment-506996235 for more info.

**Reply by kwhitefoot** [reply]
Thanks!  Works great.
